frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx

```diff
@@ -63,10 +64,12 @@ function DataSourceLister({
             }}
           >
             {fetchIconForSource(source)}
-            <p data-cy={`${String(source.name).toLocaleLowerCase().replace(/\s+/g, '-')}-add-query-card`}>
-              {' '}
-              {source.name}
-            </p>
+            <ToolTip message={source.name}>
+              <p data-cy={`${String(source.name).toLocaleLowerCase().replace(/\s+/g, '-')}-add-query-card`}>
+                {' '}
```

Comment:
can remove this

Reply:
@stepinfwd, I have removed the line no. 69 as requested. @nakulnagargade, Kindly review the same.